### PR TITLE
PR - HACKTOBERFEST

### PR DESCRIPTION
**Pull request by catcecilia**
I believe section titled "Confine adjustment and fill layers to specific areas" in https://helpx.adobe.com/photoshop/using/adjustment-fill-layers.html answers Q64

### CODE REVIEW

File: adobe-photoshop/adobe-photoshop-quiz.md [hunk 1]
```diff
@@ -500,7 +500,7 @@
 #### Q64. What is the outcome if you make a selection and then add an adjustment layer?
 
 - [ ] The adjustment layer contains a vector mask that applies the adjustment only to the unselected pixels.
-- [ ] The adjustment layer contains a layer mask that applies the adjustment only to the selected pixels.
+- [x] The adjustment layer contains a layer mask that applies the adjustment only to the selected pixels.
```
Comment (Ebazhanov) [inline adobe-photoshop/adobe-photoshop-quiz.md:503]: thanks for the answer, could you also add reference link from your PR message
```
I believe section titled "Confine adjustment and fill layers to specific areas" in https://helpx.adobe.com/photoshop/using/adjustment-fill-layers.html answers Q64
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
left comment

**Comment by catcecilia** [comment]
updated with reference link from my pr message

**Review by Ebazhanov — APPROVED** [review]
Thank you @catcecilia for your contribution and complete question! :bouquet:

**Pull request merged**